packages/pwa-kit-runtime/src/utils/ssr-namespace-paths.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * This file defines the /mobify paths used to set up our Express endpoints.
+ *
+ * If a namespace for the /mobify paths is defined, the methods in here will return the
+ * namespaced path. ie. /namespace/mobify/...
+ */
+
+const MOBIFY_PATH = '/mobify'
+const PROXY_PATH_BASE = `${MOBIFY_PATH}/proxy`
+const BUNDLE_PATH_BASE = `${MOBIFY_PATH}/bundle`
+const CACHING_PATH_BASE = `${MOBIFY_PATH}/caching`
+const HEALTHCHECK_PATH = `${MOBIFY_PATH}/ping`
+const SLAS_PRIVATE_CLIENT_PROXY_PATH = `${MOBIFY_PATH}/slas/private`
+
+const getNamespace = () => {
+    // TODO - namespaces for /mobify path will be implemented at a later date.
+    // Returns an empty string for now.
+    // Below is an example of what this implementation might look like.
+    /*
+        let {namespace = ""} = getConfig()
+        namespace = typeof namespace === 'function' ? namespace() : namespace
+        return namespace
+    */
+    return ''
+}
+
+export const startsWithMobify = (url) => {
```

Comment:
This until should probably be renamed. 

1. we shouldn't be using "mobify" in 2024
2. its not actually doing what it says. 

Lets try to come up with something that is a little more self descriptive. E.g. you are making a function that is testing if a url starts with the current namespace, so the function name should describe that. 

isGlobalNamespace
startsWithGlobalNamespace
startsWithNamespace

Are some other generic options.

Reply:
I have moved this util function into `build-remote-server.js` (the only caller of this function). 

I have also renamed this function to `_isBundleOrProxyPath`

Reply:
One thing I'd like to call out is that, based on the comments where `_isBundleOrProxyPath` we seem to be interested in checking if the path is a bundle or proxy path rather than for any `/mobify` path. So I changed the implementation a bit to check for matches on `(/namespace)/mobify/proxy/` and `(/namespace)/mobify/bundle/` rather than any `(/namespace)/mobify`.

Not sure if this is a good idea though. I'm leaning towards reverting it, but let me know what you think @bendvc